vision_statement.md

```diff
@@ -56,59 +56,9 @@ running small desktop scale simulations to test preliminary ideas
-participation from people with diverse backgrounds, skills, interests,
-and opinions. We believe that such diversity is critical in ensuring a
-growth of ideas in our community. We as a community pledge to abide by
-the following guidelines:
-
-* We pledge to treat all people with respect and provide a harassment-
-  and bullying-free environment, regardless of sex, sexual orientation
-  and/or gender identity, disability, physical appearance, body size,
-  race, nationality, ethnicity, and religion. In particular, sexual
-  language and imagery, sexist, racist, or otherwise exclusionary
-  jokes are not appropriate.
-* We pledge to respect the work of others by recognizing
-  acknowledgment/citation requests of original authors. As authors, we
-  pledge to be explicit about how we want our own work to be cited or
-  acknowledged.
-* We pledge to welcome those interested in joining the community, and
-  realize that including people with a variety of opinions and
-  backgrounds will only serve to enrich our community. In particular,
-  discussions relating to pros/cons of various technologies,
-  programming languages, and so on are welcome, but these should be
-  done with respect, taking proactive measure to ensure that all
-  participants are heard and feel confident that they can freely
-  express their opinions.
-* We pledge to welcome questions and answer them respectfully, paying
-  particular attention to those new to the community. We pledge to
-  provide respectful criticisms and feedback in forums, especially in
-  discussion threads resulting from code contributions.
-* We pledge to be conscientious of the perceptions of the wider
-  community and to respond to criticism respectfully. We will strive
-  to model behaviors that encourage productive debate and
-  disagreement, both within our community and where we are
-  criticized. We will treat those outside our community with the same
-  respect as people within our community.
-* We pledge to work from the very beginning of this project to make
-  PlasmaPy accessible to people with disabilities.
-* We pledge to help the entire community follow these guidelines, and
-  to not remain silent when we see violations of them. We will take
-  action when members of our community violate these
-  guidelines. Members of the PlasmaPy community may contact any member
-  of the Coordinating Committee to report violations. Members of the
-  Coordinating Committee will treat these reports in the strictest
-  confidence.  The Coordinating Committee will develop formal
-  procedures for how to handle reported violations.
-
-Parts of these guidelines have been adapted from the [Astropy code of
-conduct](http://www.astropy.org/about.html#codeofconduct) and the
-[Python Software Foundation code of
-conduct](https://www.python.org/psf/codeofconduct/).
+## PlasmaPy Community Code of Conduct
+
+Please read the [Code of Conduct](CODE_OF_CONDUCT.md).
```

Comment:
Perhaps a minor change would make this a little nicer:

> Please see the attached [Code of Conduct](CODE_OF_CONDUCT.md).

Reply:
👍